### PR TITLE
Remove circe-generic

### LINKED ISSUE #76: Remove circe-generic
Assign to me :)

### PR DESCRIPTION
**Pull request by kubukoz**
Closes #76.

### CODE REVIEW

File: lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/package.scala [hunk 1]
```diff
@@ -94,7 +93,7 @@ package cloudformation {
     }
   }
 
-  case class CloudFormationCustomResourceRequest[A](
+  final case class CloudFormationCustomResourceRequest[A](
       RequestType: CloudFormationRequestType,
```
Comment (kubukoz) [inline lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/package.scala:97]: IMO these should be `camelCase`, but I didn't want to tackle that in this PR.

Comment (armanbilge) [reply lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/package.scala:97]: Yeah this is @bpholt's baby, I promised him not to mess with it 😝

Comment (bpholt) [reply lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/package.scala:97]: Camel case is fine; IIRC it's only this way to make the decoder easier.

Comment (kubukoz) [reply lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/package.scala:97]: gotchu, let's follow up in another PR

**Pull request merged**